lib/PuppeteerSharp/Target.cs

```diff
@@ -68,5 +74,6 @@ public void TargetInfoChanged(TargetInfo targetInfo)
         /// <returns>A task that returns a <see cref="Session"/></returns>
         public Task<Session> CreateCDPSession() => _browser.Connection.CreateSession(TargetId);
 
+        public bool Equals(Target other) => TargetId.Equals(other.TargetId);
```

Comment:
Same here regarding the equals, are we sure about this? where are we using it?

Reply:
for `Assert.Equal` on target objects (ShouldReportWhenAServiceWorkerIsCreatedAndDestroyed)